docs/autotune/introduction.md

```diff
@@ -4,29 +4,67 @@ sidebar_label: Introduction
+Autotune automatically finds the winning variant for you among a group of candidates, and dynamically allocates traffic to optimize for a single target metric.
 
-## How Autotune works 
-Autotune executes a Bayesian multi-armed bandit algorithm to find a winning variant that optimizes for a target metric. At a high level, the multi-armed bandit algorithm works by adding more users to a treatment as soon as it recognizes that it is clearly better in maximizing the reward (the target metric). Throughout the process, higher performing treatments are allocated more traffic whereas underperforming treatments are allocated less traffic. When the winning treatment beats the second best treatment by enough margin, the process terminates. 
+## How Autotune works
+
+Autotune is Statsig's [Bayesian Multi-Armed Bandit](https://en.wikipedia.org/wiki/Multi-armed_bandit).  Autotune will test the variants you provide and measure their effect on a target metric.  As it learns, it will adjust the traffic towards the variations that perform the best until it ultimately settles on the "best" variation and divert 100% of traffic to the winner.  On the backend, we use a Thompson Sampling (Bayesian) algorithm that estimates each variant's probability of being the best variant, and allocates it that proportion of traffic.  For example, if a given variant has a 60% probability of being the best, Autotune will provide it 60% of the traffic.  At a high level, the multi-armed bandit algorithm works by adding more users to a treatment as soon as it recognizes that it is clearly better in maximizing the reward (the target metric).  Throughout the process, higher performing treatments are allocated more traffic whereas underperforming treatments are allocated less traffic. When the winning treatment beats the second best treatment by enough margin, the process terminates.
 
 ## When to use Autotune
-The main advantage of using Autotune over a regular A/B test is that Autotune generally terminates earlier than an A/B test because it requires a much smaller sample size. Autotune also suffers from fewer mistakes than A/B testing. A balanced A/B test sends 50% of traffic to each group whereas Autotune sends lesser and lesser traffic to the losing treatment. However, you want to weight the advantage of using a smaller sample size against a possibly higher false positive rate with Autotune. 
-Statsig recommends using Autotune in the following scenarios:
+
+Unlike A/B testing (Statsig Experiments), the traffic split isn't fixed over the duration of the test.  Instead, Autotune diverts more traffic to the winner and making fewer mistakes.  This is ideal when there are 4+ variations as it usually can rule out the worst performers and focus traffic on the best variants.  The downside is that users users are not guaranteed a consistent experience upon return visits.
```

Comment:
This is not a complete sentence " Instead, Autotune diverts more traffic to the winner and making fewer mistakes"


Remove duplicate "users"
"The downside is that users are not guaranteed a consistent experience upon return visits."